scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala

```diff
@@ -40,7 +41,139 @@ import scala.concurrent.Future
  */
 package object bigtable {
 
-  private val DEFAULT_SLEEP_DURATION = Duration.standardMinutes(20);
+  // =======================================================================
+  // Protobuf helpers
+  // =======================================================================
+
+  /** Helper methods for [[Mutation]]. */
+  object Mutations {
+
+    def newSetCell(family: String, column: ByteString, value: ByteString): Mutation =
```

Comment:
yeah, thats what it does but I would argue it makes more sense to default to current time since it is more natural and thats what all the higher level apis for bigtable use (the hbase-dataflow api does it if you dont add a timestamp). If somebody wants to explicitly set a timestamp to 0, they still can.